intrinsicsolver/elongateForms.go

```diff
@@ -61,7 +61,7 @@ func elongateForms(line *string, lines *[]string, idx int, name string) {
 	}
 }
 
-// SplitLinesIfNestedFunction if finds nested function then splits lines.
+// SplitLinesIfNestedFunction if finds nested function then will split lines. It's used in elongateForms().
```

Comment:
I believe pointing where the function is used is not that important, as explaining what it does and what it is used for - here we can write a short description, that the functions need to be parsed like this for the goformation parser to read properly (or sth similar)